java/src/jmri/jmrit/display/switchboardEditor/SwitchboardEditor.java

```diff
@@ -149,7 +149,7 @@
     /**
      * To count number of displayed beanswitches, this array holds all beanswitches to be displayed
      * until the GridLayout is configured, used to determine the total number of items to be placed.
-     * Accounts for "hide unconnected" setting, so it can be empty.
+     * Accounts for "hide unconnected" setting, so it can be empty. Synchronized.
      */
     private final LinkedHashMap<String, BeanSwitch> switchesOnBoard = new LinkedHashMap<>();
```

Comment:
It's recommended to annotate `switchesOnBoard` with `@GuardedBy(this)`.

See:
https://www.jmri.org/help/en/html/doc/Technical/Threads.shtml
https://www.jmri.org/help/en/html/doc/Technical/SpotBugs.shtml#annotations

Reply:
Thanks, done